tensorflow/python/ops/script_ops.py

```diff
@@ -702,12 +700,21 @@ def numpy_function(func, inp, Tout, name=None):
     inp: A list of `tf.Tensor` objects.
     Tout: A list or tuple of tensorflow data types or a single tensorflow data
       type if there is only one, indicating what `func` returns.
+    stateful: (Boolean.) If False, guarantee the runtime that
+      the function is stateless and the runtime is free to make
+      optimizations based on this, otherwise it is considered to be stateful.
```

Comment:
Suggest: "Setting this argument to False tells the runtime to treat the function as stateless, which enables certain optimizations."

Reply:
Perfect, changed